hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -293,17 +246,46 @@ public String getName() {
     return name;
   }
 
-  /**
-   * Returns Rights.
-   *
-   * @return - Rights
-   */
-  public BitSet getAclBitSet() {
-    return aclBitSet;
+  @JsonIgnore
+  public boolean isEmpty() {
+    return aclBitSet.isEmpty();
+  }
+
+  public boolean isSet(ACLType acl) {
+    return aclBitSet.get(acl.ordinal());
+  }
+
+  public boolean checkAccess(ACLType acl) {
+    return ((aclBitSet.get(acl.ordinal())
```

Comment:
Nit: let's reuse `isSet` for this

Reply:
Also can add `@VisibleForTesting` for `isSet` since it seems to be only used in tests.